tests/ConsecutiveArgumentsTest.php

```diff
@@ -5,11 +5,12 @@
 namespace Tests\MichaelPetri\PhpunitConsecutiveArguments;
 
 use MichaelPetri\PhpunitConsecutiveArguments\ConsecutiveArguments;
+use PHPUnit\Framework\Attributes\CoversClass;
 use PHPUnit\Framework\Constraint\IsAnything;
 use PHPUnit\Framework\TestCase;
 use Tests\MichaelPetri\PhpunitConsecutiveArguments\Stub\MockableConsecutiveArgumentInterface;
 
-/** @covers \MichaelPetri\PhpunitConsecutiveArguments\ConsecutiveArguments */
+#[CoversClass(ConsecutiveArguments::class)]
```

Comment:
Both PHPDoc annotations and Attributes are supported in PHPUnit 10, but annotations will generate a deprecation error in PHPUnit 11 and will no longer be supported in PHPUnit 12.

```
% ./vendor/bin/phpunit
PHPUnit 11.1.3 by Sebastian Bergmann and contributors.

Runtime:       PHP 8.3.6
Configuration: /Users/megurine/repo/php/phpunit-consecutive-arguments/phpunit.xml

.........                                                           9 / 9 (100%)

Time: 00:00.020, Memory: 4.00 MB

There was 1 PHPUnit test runner deprecation:

1) Metadata found in doc-comment for class Tests\MichaelPetri\PhpunitConsecutiveArguments\ConsecutiveArgumentsTest. Metadata in doc-comments is deprecated and will no longer be supported in PHPUnit 12. Update your test code to use attributes instead.

OK, but there were issues!
Tests: 9, Assertions: 9, Deprecations: 1.
```